core/io/resource_loader.cpp

```diff
@@ -317,55 +333,49 @@ Error ResourceLoader::load_threaded_request(const String &p_path, const String &
 		return OK;
 	}
 
-	{
-		//create load task
-
-		ThreadLoadTask load_task;
+	// Create load task.
+	ThreadLoadTask load_task;
+	load_task.requests = 1;
 
-		load_task.requests = 1;
-		load_task.remapped_path = _path_remap(local_path, &load_task.xl_remapped);
-		load_task.local_path = local_path;
-		load_task.type_hint = p_type_hint;
-		load_task.cache_mode = p_cache_mode;
-		load_task.use_sub_threads = p_use_sub_threads;
+	// REVISIT: Is it intentional that remapping is done under lock of thread_load_mutex? It does not appear to
+	// be set up under lock.
```

Comment:
I'd use `FIXME` so it pops up more easily in code searches:

```suggestion
	// FIXME: Is it intentional that remapping is done under lock of thread_load_mutex? It does not appear to
	// be set up under lock.
```

Reply:
ok thanks.  I was being too timid I guess